packages/medusa/src/models/order.ts

```diff
@@ -212,6 +213,13 @@ export class Order {
   )
   swaps: Swap[]
 
+  @Column({ nullable: true })
+  draft_order_id: string
+
+  @OneToOne(() => DraftOrder)
+  @JoinColumn({ name: "draft_order_id" })
+  draft_order: DraftOrder
+
```

Comment:
The items seem to live in the cart, not the draft order, so do these serve a purpose?